apps/src/code-studio/progress.js

```diff
@@ -175,6 +175,26 @@ progress.renderCourseProgress = function(scriptData) {
 
   const mountPoint = document.createElement('div');
   $('.user-stats-block').prepend(mountPoint);
+
+  const lessonsOverview = scriptData.showCalendar
```

Comment:
Are these lessonOverviews only used for the calendar? It might be worth calling that out in the name since I think the lesson data that shows with all the bubbles and what not comes in a different way?

Reply:
they're only for the calendar, I'll rename it

Reply:
I think it would be good to rename the prop to make it a little more clear